watertap/data/techno_economic/filter_press.yaml

```diff
@@ -26,130 +26,6 @@ default:
       value: 433972
       units: USD_2007
   removal_frac_mass_solute:
-    aluminum:
```

Comment:
Just to check - the removal of all these parameters was intended?

Reply:
Yes correct. I had initially copied the evaporation pond yaml and forgot to pare it down.